go.mod

```diff
@@ -0,0 +1,25 @@
+module github.com/rakanalh/scheduler
```

Comment:
Can you update this to `github.com/ClubNFT/scheduler`?

Reply:
Sure, I did that actually but debbuging stopped working and would not stop in the breakpoints. Weird.